### PR TITLE
Allow sorting the staff tree list page by size

### PR DESCRIPTION
**Pull request by brianhelba**

### CODE REVIEW

File: dkc/core/views.py [hunk 1]
```diff
@@ -12,14 +12,22 @@ def staff_home(request: HttpRequest) -> HttpResponse:
 
 
 @staff_member_required
-def staff_tree_size(request: HttpRequest) -> HttpResponse:
+def staff_tree_list(request: HttpRequest, sort_by) -> HttpResponse:
+    try:
```
Comment (brianhelba) [inline dkc/core/views.py:16]: @danlamanna Should I use query params here? This way is easier to build URLs with, it's just slightly annoying that I can't easily specify a default (without creating a separate URL redirect).

File: dkc/core/templates/core/staff_tree_list.html [hunk 2]
```diff
@@ -7,6 +7,22 @@ <h1 class="text-2xl font-bold text-gray-900">
       Tree Sizes
     </h1>
 
+    <div>
```
Comment (brianhelba) [inline dkc/core/templates/core/staff_tree_list.html:10]: I have no ability to style this. I tried and failed, sorry.

File: dkc/core/views.py (hunk 1 above)
Comment (zachmullen) [reply dkc/core/views.py:16]: This definitely feels more like it should be a query param. (I could see us adding more parameters in the future.)

Comment (brianhelba) [reply dkc/core/views.py:17]: Fixed.

**Review by zachmullen — APPROVED** [review]

**Pull request merged**